docs/version0.25.md

```diff
@@ -36,10 +37,46 @@ The following new features and notable changes since v 0.24.0 are included in th
 
 OpenJ9 release 0.25.0 supports OpenJDK 16. OpenJDK 16 with Eclipse OpenJ9 is *not* a long term support (LTS) release.
 
-Windows&reg; builds for Java&trade; 8 are now compiled with Microsoft&reg; Visual Studio 2013. The Visual Studio redistributable files included with the build are updated to match.
+Although it might be possible to build an OpenJDK 8 or OpenJDK 11 with OpenJ9 release 0.25.0, testing at the project is not complete and therefore support for new features that apply to these Java versions is not available.
 
 To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
 
+### ![Start of content that applies to Java 16](cr/java16.png) New JDK 16 features
+
+The following features are implemented in OpenJ9:
+
+- [JEP 338](https://openjdk.java.net/jeps/338): Vector API (incubator)
+    - The OpenJ9 project is extending this feature with optimizations.
+- [JEP 390](https://openjdk.java.net/jeps/390): Warnings for value-based classes
+- [JEP 395](https://openjdk.java.net/jeps/395): Records
+- [JEP 397](https://openjdk.java.net/jeps/397): Sealed Classes (second preview)
+
+The following features apply to OpenJ9 but are deferred to a future release:
+
+- [JEP 389](https://openjdk.java.net/jeps/389): Foreign linker API (incubator)
+- [JEP 393](https://openjdk.java.net/jeps/393): Foreign-memory access API (third incubator)
+
+The following features do not apply directly to OpenJ9 but are available in any builds of OpenJDK 16 with OpenJ9:
```

Comment:
Even though I wrote "do not apply directly" I don't like this wording, it can likely be improved. I wanted to say that OpenJDK implements these, their operation doesn't require any changes/support in OpenJ9. This probably isn't important to the end user, perhaps they should be grouped with the "features implemented by OpenJ9" which could be re-worded to "supported by OpenJ9".